galaxy/api/filters.py

```diff
@@ -93,10 +95,10 @@ def to_python_boolean(self, value, allow_none=False):
             return None
         else:
             raise ValueError(
-                u'Unable to convert "{}" to boolean'.format(unicode(value)))
+                u'Unable to convert "{}" to boolean'.format(six.text_type(value)))
```

Comment:
You can safely drop explicit casting to unicode here. Since unicode literal is used, value will be implicitly converted to unicode.